QUICKSTART.md

```diff
@@ -51,8 +51,12 @@ Span span = tracer.spanBuilder("my span").startSpan();
 try (Scope scope = tracer.withSpan(span)) {
 	// your use case
 	...
```

Comment:
```suggestion
	...
	span.setStatus(Status.OK);
```
Maybe add also this so the reader does not get confused and think status is set only on error.

Reply:
@thisthat isn't it default when we end?

Reply:
You only have to call setStatus if you want to override the default, which already is OK.

Reply:
Oh, you are right!